app/views/unattended/report_templates/host_-_products.erb

```diff
@@ -0,0 +1,40 @@
+<%#
+name: Host - Products
```

Comment:
I feel like "Installed Products" is a phrase people are used to; we use it elsewhere including the new host details Details tab. Thoughts on calling it `Host - Installed Products`?

Reply:
Sounds good to me :+1: